infrastructure/team-dev/aws/au/cluster/cluster-nodes-routing.tf

```diff
@@ -0,0 +1,31 @@
+module "routing-nodes" {
```

Comment:
@MattSurabian if you manage the eks node groups separately from the cluster you can attach the security groups (e.g. ingress) directly to specific nodes. I think it's possible if you manage them in the main module too but I couldn't figure out how to make it work and just bailed to this.